versions.properties

```diff
@@ -43,6 +43,7 @@ net.sf.ehcache:ehcache-core=2.6.0
 saxonVersion=9.5.1-8
 net.sf.saxon:Saxon-EE=$saxonVersion
 abderaVersion=1.1.3
+org.asciidoctor:asciidoctorj-diagram=1.5.0
```

Comment:
It was inserted alphabetically. Fair enough, though, I'll move it after the apache stuff.